SI/Sem.05/Pract.05/Task03.cpp

```diff
@@ -0,0 +1,28 @@
+#include <iostream>
+using namespace std;
+
+void intToBinary(int num)
```

Comment:
We need to return the number.

We convert it to binary and return it.

Revise this task.